internal/terraform/context_plan_test.go

```diff
@@ -4536,7 +4536,15 @@ func TestContext2Plan_ignoreChanges(t *testing.T) {
 func TestContext2Plan_ignoreChangesWildcard(t *testing.T) {
 	m := testModule(t, "plan-ignore-changes-wildcard")
 	p := testProvider("aws")
-	p.PlanResourceChangeFn = testDiffFn
+	p.PlanResourceChangeFn = func(req providers.PlanResourceChangeRequest) (resp providers.PlanResourceChangeResponse) {
+		// computed attributes should not be set in config
```

Comment:
What about Optional+Computed?

Reply:
If it has a value in config (then we do not need to check if it is Optional), then we need to return the prior value

Reply:
Thanks! Turns out it's actually easier to just check for !optional, regardless of whether there's a config value or not. Since there is no way of knowing whether a prior optional+computed value came from config, the only thing we can do here is return the prior value.